### PR TITLE
7397 docs getting started

### PR DESCRIPTION
**Pull request by denpira**
Collapse/expand & hyperlinks.
https://codio.myjetbrains.com/youtrack/issue/codio-7397.

### CODE REVIEW

File: source/stylesheets/pages/_docs.scss [hunk 1]
```diff
@@ -396,3 +399,53 @@ body.docs .crumbs {
...
+  font-family: "Glyphicons Halflings";
+  content: "\e114";
+  //float: right;
+  margin-left: -6px;
+}
+
+/* Icon when the collapsible content is hidden */
+/*.btn.collapsed:after {
+  content: "\e080";
+}*/
```
Comment (mloginov) [inline source/stylesheets/pages/_docs.scss]: Maybe it should be removed or we will need it in future?

File: source/stylesheets/pages/_docs.scss [hunk 2]
```diff
@@ -396,3 +399,53 @@ body.docs .crumbs {
...
+
+body.docs .btn {
+  background: #FFF;
+  //font-size: 1rem;
```
Comment (mloginov) [inline source/stylesheets/pages/_docs.scss]: remove?

Comment (denpira) [inline source/stylesheets/pages/_docs.scss]: remove

**Comment by mloginov** [comment]
Ok for me :+1:

**Pull request merged**